jekyll/_includes/v2-banner.html

```diff
@@ -0,0 +1,3 @@
+<div class="alert alert-info" role="alert">
+Refer to <a href="http://circleci.com/2.0/overview/">CircleCI 2.0 documentation for administrators</a> to get started with installing or upgrading your installation to 2.0.
```

Comment:
Should this maybe point directly to the Administration intro?

Reply:
overview is the admin intro